.buildkite/steps/copy.sh

```diff
@@ -110,15 +112,16 @@ if [[ $BUILDKITE_BRANCH != "master" ]] && [[ "$BUILDKITE_TAG" != "$BUILDKITE_BRA
   cat << EOF > "$mapping_file"
 Mappings:
   AWSRegion2AMI:
-    ${AWS_REGION} : { linux: $linux_source_image_id, windows: $windows_source_image_id }
+    ${AWS_REGION} : { linux: $linux_amd64_source_image_id, linuxarm: $linux_arm64_source_image_id, windows: $windows_amd64_source_image_id }
 EOF
   exit 0
 fi
```

Comment:
From this point on, the copy.sh script doesn't run on non-master branches. I haven't tested that copying all AMIs from us-east-1 to other region works.